roles/finch/tasks/main.yml

```diff
@@ -19,19 +22,31 @@
       unarchive:
         dest: '{{ finch.install_path }}'
         src: '{{ finch.zip }}'
+        owner: root
+        group: root
+        mode: 0755
   when: st.stat.checksum|default("") != finch.hash
 - name: Install Finch udev rule
   copy:
     src: 55-finch.rules
     dest: '/etc/udev/rules.d/'
+    owner: root
+    group: root
+    mode: 0644
 - name: Install Finch extra - dance2.py
   copy:
     src: dance2.py
     dest: '{{ finch.install_path }}'
+    owner: root
+    group: root
+    mode: 0755
```

Comment:
Without the shebang, these aren't actually executable, so I think I was mistaken in telling you 0755 and it should probably be "0644". Thoughts @kylelaker?

Reply:
Any Python module that won't be executed (via a call to `execve(2)` or friends) does not need `0755` and should be `0644`. If a Pythin file has a hashbang or `if __name__ == '__main__'`, it's probably intended to be `execve(2)`-ed. Without a hashbang and with `0755` if you were to `./dance2.py`, bash would run it as bash which isn't ideal. We should make sure of this across all the Finch scripts.